components/fileselect/initial-files.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Initial Files
+page_title: FileSelect Initial Files
+description: The FileSelect allows you to define a collection of pre-selected files, which will be initially shown in the selected file list.
+slug: fileselect-initial-files
+tags: telerik,blazor,fileselect,initial, files
+published: True
+position: 40
+---
+
+# FileSelect Initial Files
+
+The `Files` parameter of the FileSelect accepts an `IEnumerable<FileSelectFileInfo>` collection that stores a set of pre-selected files. It allows you to display files in the file list initially, making it convenient for showcasing previously uploaded files.
```

Comment:
Try to follow this pattern when writing the introduction for a feature article:
1. What is the feature? -> Showing files on initial load.
2. Why do I need it? -> To show previously uploaded files.
3. How do I use it? -> By using the  `Files` parameter.

```suggestion
The Blazor FileSelect component enables you to display specific files in the file list when the component loads for the first time. This is a convenient way to show previously uploaded files.

To configure the initially displayed files, use the `Files` parameter of the FileSelect—it accepts an `IEnumerable<FileSelectFileInfo>` collection that stores a set of pre-selected files.
```